packages/build/tests/install/snapshots/tests.js.md

```diff
@@ -846,68 +779,6 @@ Generated by [AVA](https://avajs.dev).
     (Netlify Build completed in 1ms)␊
     Build step duration: Netlify Build completed in 1ms`
 
-## Install local plugin dependencies: propagate errors
```

Comment:
I was getting a weird test failure in MacOS, so I got rid of the snapshot.

Reply:
Suffering same issue, thanks for this

Reply:
@lemusthelroy can I get a ✅? I lost Simon's when I updated the tests.